### PR TITLE
Add training and career

### PR DESCRIPTION
**Pull request by wcrum**
Known issues:
- [ ] First two content images are hidden when in mobile. Adjust css to support mobile.

### CODE REVIEW

File: src/pages/career.astro [hunk 1]
```diff
@@ -1,66 +1,97 @@
...
+  >
+    <div
+      class="relative z-20 h-fit py-20 bg-cover bg-no-repeat bg-center"
+      style={{
+        backgroundImage: `url('/images/soldier_whiteboard.png')`,
+      }}
+    >
       <Hero
-        title="WE ARE THE MARINE CORPS SOFTWARE FACTORY"
+        title="PICK THE PERFECT ROLE FOR YOU"
```
Comment (dfullaway) [inline src/pages/career.astro]: This should say "WE ARE THE MARINE CORPS SOFTWARE FACTORY"

File: src/pages/career.astro [hunk 2]
```diff
@@ -1,66 +1,97 @@
...
+        content="Lorem ipsum dolor sit amet. Et quae quibusdam sed consequatur iste nam asperiores facilis qui error iste qui quia tenetur est rerum dolorem. Eum eveniet placeat cum impedit officiis est odit asperiores.  Lorem ipsum dolor sit amet. Et quae qui"
         link="#"
-        linkText="LEARN MORE"
+        linkText="APPLY NOW"
         padding="pt-24"
+        titleClasses="text-white"
```
Comment (dfullaway) [inline src/pages/career.astro]: Change text size to 64px.

File: src/pages/career.astro [hunk 3]
```diff
@@ -1,66 +1,97 @@
...
+      <div class="hidden lg:block">
+        <div class="relative z-20">
+          <ContentImageFull
+            title="BUILD SOFTWARE"
+            content="Software is developed using Agile methodology in teams. Teams consist of a Product Manager (PM), a user interface/user experience (UI/UX) designer, and software developers."
+            link=""
+            linkText=""
+            titleClasses="!text-white text-2xl font-bold md:text-3xl lg:text-4xl"
+            imgSrc="/images/charlie-1.png"
+            imgAltText="as"
```
Comment (dfullaway) [inline src/pages/career.astro:52]: Content font size should be 2xl.

File: src/pages/career.astro [hunk 4]
```diff
@@ -1,66 +1,97 @@
...
+
+    <div class="relative z-20 max-h-screen scrollbar-hide">
+      <div class="hidden lg:block">
+        <div class="relative z-20">
+          <ContentImageFull
+            title="BUILD SOFTWARE"
+            content="Software is developed using Agile methodology in teams. Teams consist of a Product Manager (PM), a user interface/user experience (UI/UX) designer, and software developers."
+            link=""
+            linkText=""
+            titleClasses="!text-white text-2xl font-bold md:text-3xl lg:text-4xl"
```
Comment (dfullaway) [inline src/pages/career.astro:50]: Title text should be 40 px.

File: src/pages/career.astro
Comment (dfullaway) [inline src/pages/career.astro:1]: Missing the "See job listings" button and below. Most of the content is gone in mobile.

**Review by dfullaway — COMMENTED** [review]
It looks like you've only got the top half of the page done. Also, the weirdness with the disappearing content needs to get fixed. We may also need to start merging some stuff into the 15-Apr-Mas-updates branch, as I suspect we are generating merge conflicts.

File: src/pages/career.astro
Comment (wcrum) [reply src/pages/career.astro:1]: Fixed mobile content.

**Pull request merged**